_build/buildPDF.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# this generates A PDF from a HTML file
+
+if [ ! -x _build/build.sh ] ; then
+  echo "ERROR: script must be run in root of docs dir"
```

Comment:
This is the same as the start of build.sh, I think we should keep all scripts in a similar format if possible